### PR TITLE
feat(react): add `Skeleton` & `InputAdornment` components

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose

Wrap `Skeleton` & `InputAdornment`  components from MUI.

### Related Issues
- https://github.com/wso2/oxygen-ui/issues/2

### Related PRs
- N/A

### Checklist
- [ ] UX/UI review done on the final implementation.
- [ ] Story provided. (Add screenshots)
- [ ] Manual test round performed and verified.
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Documentation provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: packages/react/src/components/Skeleton/Skeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (pavinduLakshan) [inline packages/react/src/components/Skeleton/Skeleton.tsx:2]:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

File: packages/react/src/components/Skeleton/__test__/Skeleton.test.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (pavinduLakshan) [inline packages/react/src/components/Skeleton/__test__/Skeleton.test.tsx:2]:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

File: packages/react/src/components/Skeleton/index.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (pavinduLakshan) [inline packages/react/src/components/Skeleton/index.ts:2]:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

**Review by pavinduLakshan — DISMISSED** [review]

File: packages/react/src/components/Skeleton/Skeleton.tsx (hunk 1 above)
Comment (brionmario) [reply packages/react/src/components/Skeleton/Skeleton.tsx:2]: This should be updated in the WSO2 ESLint Plugin.

cc:/ @savindi7

File: packages/react/src/components/Skeleton/__test__/Skeleton.test.tsx (hunk 2 above)
Comment (brionmario) [reply packages/react/src/components/Skeleton/__test__/Skeleton.test.tsx:2]: This should be updated in the WSO2 ESLint Plugin.

cc:/ @savindi7

File: packages/react/src/components/Skeleton/index.ts (hunk 3 above)
Comment (brionmario) [reply packages/react/src/components/Skeleton/index.ts:2]: This should be updated in the WSO2 ESLint Plugin.

cc:/ @savindi7

**Review by ShanChathusanda93 — APPROVED** [review]

**Pull request merged**